docs/devel/contributing.md

```diff
@@ -2,74 +2,91 @@
 
 This guide will guide you through the process of setting up a development environment and contributing to Nuclio. 
 
-## Set up some prerequisites
-Obviously, you'll need:
+#### In This Document
+- [Prerequisites](#prerequisites)
+- [Getting the source code](#get-source)
+- [Setting up a GoLand project](#goland-setup)
+- [Submitting a PR](#submitting-a-pr)
+
+<a id="prerequisites"></a>
+## Prerequisites
+
+Ensure that your setup includes the following prerequisite components:
+
 - Linux or OSX
-- git
-- Docker (version 17.05+, because Nuclio uses [multi-stage builds](https://docs.docker.com/engine/userguide/eng-image/multistage-build/))
+- Git
+- Docker version 17.05+ (because Nuclio uses [multi-stage builds](https://docs.docker.com/engine/userguide/eng-image/multistage-build/))
 - The Go toolchain (CI tests with 1.10, best use that)
-- A `GOPATH` directory and `GOPATH` environment variable set to that
-- Kubernetes 1.7+ (for testing, mostly) - `minikube` recommended (you can follow the [minikube getting started guide](/docs/setup/minikube/getting-started-minikube.md))
+- A **GOPATH** directory and a `GOPATH` environment variable that's set to that directory
+- Kubernetes version 1.7+ (mostly for testing) - `minikube` recommended; (you can follow the [Minikube getting-started guide](/docs/setup/minikube/getting-started-minikube.md))
+
+<a id="get-source"></a>
+## Getting the source code
 
-## Getting the source
-Fork Nuclio and clone it:
+Fork the Nuclio GitHub repository and clone it:
 
 ```sh
 git clone https://github.com/<your username>/nuclio.git $GOPATH/src/github.com/nuclio/nuclio
 ```
 
-Check out the development branch, because that's where all the goodness happens:
+Check out the `development` branch, because that's where all the goodness happens:
 
 ```sh
 cd $GOPATH/src/github.com/nuclio/nuclio && git checkout development
 ```
 
-Now `go get` some dependencies that are injected into functions and cannot be vendored:
+Now, use `go get` to get some dependencies that are injected into functions and cannot be vendored:
 
 ```sh
 go get github.com/v3io/v3io-go-http/... && go get github.com/nuclio/logger/... && go get github.com/nuclio/nuclio-sdk-go/... && go get github.com/nuclio/amqp/... && go get github.com/v3io/scaler-types/...
 ```
 
-Build Nuclio artifacts (`nuctl`, Docker images):
+Build Nuclio artifacts (`nuctl`, container images):
```

Comment:
what is wrong with Docker images?

Reply:
Both terms are OK, but we wanted to unify our terminology (which was inconsistent) and in consultation with Yaron H. and Oded we decided to use "container images" — which is more generic and isn't restricted to Docker; (currently, we only supported Docker as a container platform, but this might change in the future). We explain in the intro that we support Docker. See PR #1326.